Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get -q update
 RUN DEBIAN_FRONTEND=noninteractive \
     apt-get -q -y dist-upgrade
 RUN DEBIAN_FRONTEND=noninteractive \
-    apt-get -q -y install software-properties-common
+    apt-get -q -y install curl software-properties-common
```

Comment:
It's a command-line HTTP client. It also supports a lot of other protocols. A lot of people use it to download files from web servers on the command-line or to test REST APIs that use JSON. The curl website is: http://curl.haxx.se.